mysql-test/suite/tianmu/t/create_index.test

```diff
@@ -0,0 +1,5 @@
+create table t1 (f1 binary(5)) engine=innodb;
```

Comment:
The test case lacks the description of the test module.

Reply:
ok